MvvmCross/Core/Binding/Bindings/Target/Construction/MvxTargetBindingFactoryRegistry.cs

```diff
@@ -116,15 +116,16 @@ private IMvxPluginTargetBindingFactory FindSpecificFactory(Type type, string nam
             {
                 return factory;
             }
+            var baseType = type.GetTypeInfo().BaseType;
+            if (baseType != null)
+                factory = this.FindSpecificFactory(baseType, name);
+            if (factory != null) return factory;
```

Comment:
In my case, it allow me to add a click binding to the IMenuItem, aware of ICommand.CanExecute.

`            registry.RegisterFactory(new MvxCustomBindingFactory<IMenuItem>("Click", menuItem => new MenuItemClickBinding(menuItem)));
`

`            set.Bind(SalvarMenuItem)
                .For("Click")
                .To(vm => vm.SalvarCommand);
`

You don't find a factory because the class searched for a factory is Android.Views.IMenuItemInvoker, which is an internal class, and you cann't register a factory to it.

Then you register a factory to IMenuItem (which is correct).